packages/components/src/Checkbox/Checkbox.test.tsx

```diff
@@ -19,6 +20,23 @@ it("renders a disabled Checkbox", () => {
   expect(tree).toMatchSnapshot();
 });
 
+it("renders a checkbox with an empty element", () => {
+  const tree = renderer
+    .create(
+      <Checkbox>
+        <></>
+      </Checkbox>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
interesting test. a fragment is technically treated the same as any react element like `<Text />` so, not too sure if you should actually test it.

If you really want to test it, you can avoid that snapshot and add a `data-testid` on the label span and check via `expect(getByTestId("YOR_TEST_ID").children).tohaveLength(0)`if it has a child

Reply:
yeah, I first suggested testing the empty element as it was my first impression that it might be an edge case,  but after a second thought, I think it's eventually the same as other elements like `<Text />`. It was my bad to suggest that in the first place, so I will leave it to you @darryltec @ChristianLee-Jobber to decide whether to keep it or not. But I totally agree with @darryltec, it's the same as other elements

Reply:
Thanks for bringing that up @darryltec. I have removed the test